api/app/routers/sfms.py

```diff
@@ -0,0 +1,61 @@
+""" Router for SFMS """
+import os
+import io
+import logging
+from datetime import datetime
+from tempfile import SpooledTemporaryFile
+from fastapi import APIRouter, UploadFile, Response, Request
+from app.utils.s3 import get_client
+from app import config
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix="/sfms",
+)
+
+
+class FileLikeObject(io.IOBase):
```

Comment:
Why this wrapper class? It's wrapping the exact same method signatures underneath?

Reply:
I've added a comment in the code: "The aiobotocore library expects a file-like object, but we can't pass the SpooledTemporaryFile
    object directly to aiobotocore. aiobotocore looks for a "tell" method, which isn't present
    on SpooledTemporaryFile. aiobotocore doesn't need an object with a tell method, and understands
    how to use IOBase, so we can wrap the SpooledTemporaryFile in a class that implements IOBase
    to make aiobotocore happy."

Reply:
Ok thanks, that's good to know. Do you still need to push up the commit with the comment?